projects/go-lib/src/lib/components/go-button/go-button.component.scss

```
@@ -54,7 +54,7 @@ $button-shadow-secondary-active: 0 0 0 3px transparentize($theme-light-bg-hover,
   }
 
   &--icon-only {
-    padding: calc(.625rem - 2px) .625rem;
+    padding: calc(.625rem - 2px) 1rem;
```

@Pooja-kj   Can we remove the `1rem` padding which is added to the left and right as it doesn't make any change?
```suggestion
    padding: calc(.625rem - 2px);
```